### ISSUE TITLE
Env var to bypass Prisma version validation 

### ISSUE THREAD

**Issue by divyenduz** [MichalLytek/typegraphql-prisma#31]
We are writing an E2E test with `dev` version of Prisma and `typegraphql-prisma`. The goal is to detect any breaking changes from Prisma's end in `typegraphql-prisma` as early as possible. 

Today this is not possible because validation returns 

```
Error: Looks like you use an incorrect version of the Prisma packages: "2.12.0-dev.8". Please ensure that you have installed a version that meets 'typegraphql-prisma' requirement: "~2.11.0".
```

It would be great to have an env var like `VALIDATE_PRISMA_VERSION=<boolean>` to bypass this check.

**Comment by MichalLytek** [comment]
Closed by d8f31e8 🔒

**Comment by divyenduz** [comment]
Awesome, thank you :)
When is this released?

**Comment by MichalLytek** [comment]
I hope today, I need to fix a few other issues 😉

**Comment by MichalLytek** [comment]
Released as v0.8.5 🚀 
https://github.com/MichalLytek/typegraphql-prisma/releases/tag/v0.8.5
